packages/atlas/lib/src/screen_coordinate.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/foundation.dart';
+/// A pair of x and y coordinates.
+/// The `x` and `y` coordinates are stored as screen pixels (not display pixels)
+/// relative to the top of the map, not top left of the whole screen.
+class ScreenCoordinate {
+  /// The x coordinate in screen pixels.
+  final int x;
+
+  /// The y coordinate in screen pixels.
+  final int y;
+
+  const ScreenCoordinate({
+    @required this.x,
+    @required this.y,

```

Comment:
I know that in other classes the constructor has a slightly different approach ("@required int x, f.e). For me this ended in a compiler error, so I went with that approach. Feel free to let me know, how I could change it so that it works with your CodingGuidelines.